src/libraries/icubmod/usbCamera/linux/V4L_camera.cpp

```diff
@@ -90,6 +90,13 @@ V4L_camera::V4L_camera() : RateThread(1000/DEFAULT_FRAMERATE), doCropping(false)
     param.camModel = SEE3CAMCU50;
     myCounter = 0;
     timeTot = 0;
+    configFx = true;
```

Comment:
I'd initialize it to false and set the flag to true when the param is actually found.
_The result is the same_ as long as the code is bug free, but it looks more safe to me in case any bug will be introduced in the future.
The idea is : _assuming theere is a bug somewhere_, it is safer a device that says "I don't have this parameter" when the parameter is correctly loaded, then a device that says "I have this parameter" when it actually don't.